CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 All notable changes to this project will be documented in this file.
 
+## [2.1.0] - 2023-11-03
+- The default container name has been updated to `localstack-main`, but we still manage containers with the name `localstack_main`
```

Comment:
I think this sentence might be a bit technical / confusing for users who just use the orb.
Maybe we should enclose the technical details in a top level item like this?
```suggestion
- Preparing the Orb for the release of LocalStack 3.0.
  - The default container name will be renamed from `localstack-main` to `localstack_main`.
  - This change is backwards compatible, it will fall back to the old container name for now (and will use the new container name once LocalStack 3.0 is released).
```